backend/infrastructure/network/getWorkflowStatus.ts

```diff
@@ -8,7 +8,6 @@ import { getCleanStatus } from '../../application/getCleanStatus';
 export async function getWorkflowStatus(input: RepoInput): Promise<GitHubStatus> {
   const { owner, repo, ref } = input;
   const token = process.env.GITHUB_TOKEN || '';
-  
```

Comment:
We can leave this be, but here the newline is to make it easier to read and understand the sections of the code - we have dumb, starter assignments up top, and the dynamic assignment is separated optically. Because it does a lot more, it makes it more understandable how it's put together when spaced away slightly from the trivial stuff. No real difference either way, but it makes the semantics a bit clearer in my opinion - it just reads better.

But I am OK with leaving this change - you choose!

Reply:
It wasn't there originally, I added it in one of my commits 🛩️ 

I'll throw it back in.